SPECS/ca-certificates/ca-certificates.spec

```diff
@@ -189,6 +189,14 @@ Requires: %{name}-shared = %{version}-%{release}
 Provides a legacy version of ca-bundle.crt in the format of "[hash].0 -> [hash].pem"
 pairs under /etc/pki/tls/certs.
 
+%package static
+Summary:  ca-certs package with minimized runtime requirements
+Group:    System Environment/Security
+Requires: %{name}-shared = %{version}-%{release}
+
+%description static
+Provides ca-certificates while minimizing the runtime dependencies. This package does not contain the post section.
```

Comment:
The `%post` section is what looks at the sources and puts the certificates in the right places for them to be usable to different system components. I'm not sure what's the purpose of this particular subpackage.